test/lib/amqp-util.test.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const chai = require('chai')
```

Comment:
Please, follow the Eslint rules enforced for the project. As we didn't have tests before, currently the `gulp eslint` task is't linting the new `test` directory.

But if you update `gulpfile.js:L17` with: 

```js
gulp.src([].concat(config.paths.src, config.paths.test))
```

and run `gulp eslint`, you'll see a total of ✖ 79 problems (75 errors, 4 warnings).

Could you fix those for me, please?